zulip_bots/zulip_bots/bots/lunch/test_lunch.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+from zulip_bots.test_lib import StubBotTestCase
+
+class TestLunch(StubBotTestCase):
+    bot_name = 'lunch'
+
+    def test_bot(self):
+        msg = dict(
```

Comment:
I know you copied this style from another test, but it's not clear to me that some of the items in this dictionary make the test more readable.  I think it's better if we can read the tests and clearly see how the food items get added and removed.